From 79faa0ebb51dbbf2f6fe6918fb83dc9e1f6605f4 Mon Sep 17 00:00:00 2001
From: Geert Uytterhoeven <geert@linux-m68k.org>
Date: Tue, 18 Nov 2008 20:54:05 +0100
Subject: [PATCH] m68k: Atari SCC serial driver needs atari_SCC_init_done

commit 5575d0a3c9676b2886adad67dd4b2ac126a49f1f ("m68k/atari/debug.c: possible
cleanups") removed atari_SCC_init_done, which we now need again

Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 arch/m68k/atari/debug.c |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/arch/m68k/atari/debug.c b/arch/m68k/atari/debug.c
index 28efdc3..24dd582 100644
--- a/arch/m68k/atari/debug.c
+++ b/arch/m68k/atari/debug.c
@@ -20,6 +20,10 @@
 #include <asm/atarihw.h>
 #include <asm/atariints.h>
 
+/* Flag that Modem1 port is already initialized and used */
+int atari_SCC_init_done;
+EXPORT_SYMBOL(atari_SCC_init_done);
+
 /* Can be set somewhere, if a SCC master reset has already be done and should
  * not be repeated; used by kgdb */
 int atari_SCC_reset_done;
@@ -259,6 +263,7 @@ static void __init atari_init_scc_port(int cflag)
 	SCC_WRITE(5, reg5 | 8);
 
 	atari_SCC_reset_done = 1;
+	atari_SCC_init_done = 1;
 }
 
 static void __init atari_init_midi_port(int cflag)
-- 
1.5.6.5

